tests/test-cases/Group1-Docker-Commands/1-25-Docker-Port-Map.robot

```diff
@@ -12,9 +12,11 @@ Create container with port mappings
     ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} start webserver
     Should Be Equal As Integers  ${rc}  0
     Should Not Contain  ${output}  Error
-    ${rc}  ${output}=  Run And Return Rc And Output  wget ${vch-ip}:10000 --retry-connrefused --timeout=10 --tries=10
```

Comment:
Good call, @dougm. I'll refactor this test. `wget` doesn't retry if it gets a `connection refused` error unless `--retry-connrefused` is specified. I'll use robot's wait and retry mechanism instead.

@mhagen-vmware, nginx takes just a few seconds to come up when I run this test locally. `start` maps the ports and commits the container, and there could be latency/network issues in some cases.